File: PolicyManagement.Api/PolicyManagement.Dtos/Customer/CustomerShortDetailDto.cs
using System;

namespace PolicyManagement.Dtos.Customer
{
    public class CustomerShortDetailDto
    {
        public string NameInPolicy { get; set; }
        public string AddressInPolicy { get; set; }
        public string CustomerCode { get; set; }
        public string ClusterCode { get; set; }
        public string Email { get; set; }
        public string Cluster { get; set; }
        public string CustomerType { get; set; }
        public string ContactPerson { get; set; }
        public string ContactNumber { get; set; }
        public string Pan { get; set; }
        public string Gstin { get; set; }
        public int CustomerId { get; set; }
        public string Gender { get; set; }
        public string DateOfBirth { get; set; }
        public string PassportNumber { get; set; } 
        public Nullable<short> GenderId { get; set; }
        public int? ClusterId { get; set; }
        public string AadhaarNo { get; set; }
        public string Profession { get; set; }
        public int? CityId { get; set; }
        public int? ReferById { get; set; }
        public int? ReferenceId { get; set; }
        public int? PosId { get; set; }
        public int? TeamMemberId { get; set; }


    }
}

File: PolicyManagementAngular/src/app/app-entites/dtos/customer/customer-short-detail-dto.ts
export interface ICustomerShortDetailDto
{
    NameInPolicy: string;
    AddressInPolicy: string;
    CustomerCode: string;
    ClusterCode: string;
    Email: string;
    Cluster: string;
    CustomerType: string;
    ContactPerson: string;
    ContactNumber: string;
    Pan: string;
    Gstin: string;
    Gender: string;
    DateOfBirth: string;
    PassportNumber: string;
    CustomerId: number;
    ClusterId?:number,
    Profession :  string;
    Name? :  string;
    CityId? :  number;
    ReferById? :  number;
    ReferByName? :  string;
    PosId? :  number;
    TeamMemebrId? :  number;
    ReferenceId? :  number;

}

export interface ICustomerInsuranceDetail{
    Name: string;
    Address: string;
    CustomerCode: string;
    Code: string;
    Email: string;
    Cluster: string;
    CustomerType: string;
    ContactPerson: string;
    Mobile: string;
    Pan: string;
    Gstin: string;
    Gender: string;
    DateOfBirth: Date;
    PassportNumber: string;
    CustomerId: number;
    ClusterId?:number
    Profession :  string;
    RelationProposer?: number;
    SumInsuredIndividual?: number;
    SumInsuredFloater?: number;
    CumulativeBonus?: string;
    Deductable?: string;
    LoadingReason?: string;
    Loading?: string;
    Ped?: string;
    PedExclusion?: string;
    Ppc?: string;
    AnualIncome?: string;
    RiskClass?: number;
    NomineeName?: string;
    NomineeRelationship?: string;
    Aadhar?: string;
    GenderId:number;
    BranchId:number;
    CityId:number;
    uid:number;
    ReferById? :  number;
    PosId? :  number;
    TeamMemebrId? :  number;
    ReferenceId? :  number;
    NomineeRelationShipName? :  string;
    PedName? :  string;
    PpcName? :  string;
    RelationProposerName?: string;
    ProfessionName?: string;
    Age?: Number;
}